src/features/fireWeather/components/graphs/WxDataGraphToggles.tsx

```diff
@@ -47,7 +50,10 @@ const WxDataToggles = ({
   setShowForecasts,
   noPastForecasts,
   showPastForecasts,
-  setShowPastForecasts
+  setShowPastForecasts,
+  noRecentHistoricModels,
+  showRecentHistoricModels,
+  setShowRecentHistoricModels
```

Comment:
I don't think we need to create these and pass down here, we can just use showModelSummaries. I will make this change